flake.lock

```diff
@@ -0,0 +1,41 @@
+{
+  "nodes": {
+    "flake-utils": {
+      "locked": {
+        "lastModified": 1623875721,
+        "narHash": "sha256-A8BU7bjS5GirpAUv4QA+QnJ4CceLHkcXdRp4xITDB0s=",
+        "owner": "numtide",
+        "repo": "flake-utils",
+        "rev": "f7e004a55b120c02ecb6219596820fcd32ca8772",
```

Comment:
I have no clue about nix, but with hashes and dates in this file, does it mean it has to be updated regularly?

Reply:
@andreasabel No, flake-utils is very simple and unlikely to be needed to be updated, and nixpkgs is only used for convenience. We could get rid of the nixpkgs usage and still provide most of the value of this file.